sunpy/net/dataretriever/sources/fermi_gbm.py

```diff
@@ -0,0 +1,104 @@
+from urllib.parse import urljoin
+from sunpy.util.scraper import Scraper
+from ..client import GenericClient
+
+__all__ = ['GBMClient']
+
+
+class GBMClient(GenericClient):
```

Comment:
Can you add a docstring to the class here describing what Fermi is, what the data products are and the options available through Detector and Resolution?